hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -0,0 +1,58 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "nuvoton:top_earlgrey:ast:0.1"
```

Comment:
we should rename this `lowrisc:systems:ast`, since this is open-source after all.
I am also starting to use more generic names, ie `systems` instead of `top_earlgrey` for the open source side.

Since this will make things more portable with future tops (just switch the cores-root)